less/forms.less

```diff
@@ -729,7 +729,7 @@ input[type="checkbox"] {
     }
 }
 :not(.has-error).focus {
-    > label {
```

Comment:
I don't think we need to keep the regular `label` styling here, can improve specificity by just having `.control-label`.